examples/minioinstance-with-external-service.yaml

```diff
@@ -30,24 +37,37 @@ spec:
       prometheus.io/path: /minio/prometheus/metrics
       prometheus.io/port: "9000"
       prometheus.io/scrape: "true"
-  ## MinIO release version, (minio/minio image tag).
-  version: RELEASE.2019-07-10T00-34-56Z
+  ## Registry location and Tag to download MinIO Server image
+  image: minio/minio:RELEASE.2019-07-24T02-02-23Z
   ## Secret with credentials to be used by MinIO instance.
   credsSecret:
     name: minio-creds-secret
+  ## Supply number of replicas.
+  ## For standalone mode, supply 1. For distributed mode, supply 4 or more (should be even).
+  ## Note that the operator does not support upgrading from standalone to distributed mode.
+  replicas: 4
+  ## Enable Kubernetes based certificate generation and signing as explained in
+  ## https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster
+  requestAutoCert: true
+  ## Used when "requestAutoCert" is set to true. Set the organization, commonName and
+  ## validity duration for the auto-generated certificate.
+  certConfig:
+    organization: []
+    commonName: ""
+    duration: 365
   ## Add environment variables to be set in MinIO container (https://github.com/minio/minio/tree/master/docs/config) 
   env:
     - name: MINIO_BROWSER
       value: "on"
-    - name: MINIO_STORAGE_CLASS_RRS
-      value: "EC:2"
+    # - name: MINIO_STORAGE_CLASS_RRS
+    #   value: "EC:2"
   ## Configure resource requests and limits for MinIO containers
   resources:
     requests:
-      memory: 256Mi
+      memory: 512Mi
       cpu: 250m
     limits:
-      memory: 512Mi
+      memory: 8192Mi
```

Comment:
We shouldn't use limits due to bugs in CFS implementation Linux kernel. https://github.com/kubernetes/kubernetes/issues/67577#issuecomment-516661131